src/docker-outside-of-docker/devcontainer-feature.json

```diff
@@ -20,6 +20,11 @@
             "default": true,
             "description": "Install OSS Moby build instead of Docker CE"
         },
+        "mobyBuildxVersion": {
+            "type": "string",
+            "default": "0.12.0",
```

Comment:
Are we planning to reset it to latest once the upstream issue is fixed, or are we going to deterministically pick and maintain the default here?

Reply:
Saw it's covered in https://github.com/devcontainers/features/issues/839
Cool